### PR TITLE
Emotion should forward innerRef

### PR DESCRIPTION
**Pull request by tkh44**
Per this twitter conversation, https://twitter.com/colebemis/status/1009909682889035776, system-components was not passing innerRef through. This *should* fix it.

@mitchellhamilton or @Andarist might know of a better way

### CODE REVIEW

File: system-components/src/emotion.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import styled from 'react-emotion'
 import System from './System'
 
 const create = new System({
-  createComponent: type => (...args) => styled(type)(...args)
+  createComponent: type => (...args) =>
+    styled(type, { shouldForwardProp: name => name === "innerRef" })(...args)
```
Comment (mitchellhamilton) [inline system-components/src/emotion.js]: It should be this.
```jsx
name => name !== 'ref' && name !== 'theme'
```

Comment (colebemis) [reply system-components/src/emotion.js]: Nice! I tried that out in a code sandbox and it worked 🎉 https://codesandbox.io/s/4qvpyo8r09

@jxnblk Looks like we didn't need to change `clean-tag` after all :)

**Comment by tkh44** [comment]
This has been updated per @mitchellhamilton's fix [https://github.com/jxnblk/styled-system/pull/219/files#r197373080]

Should be good to go 👍

**Comment by jxnblk** [comment]
Awesome, thanks!

**Pull request merged**